src/pages/authorization/faq/index.mdx

```diff
@@ -56,6 +56,15 @@ curl 'https://accounts.livechat.com/customer/' \
 
 It's the same request as the one to [**create a new customer**](/authorization/customer-authorization#case-new-customer). The only difference it that it has an additional parameter, `entity_id`.
 
+## HelpDesk authorization
+
+#### How can I authorize within HelpDesk?
```

Comment:
I'm not sure about "within". Before you authorize in HelpDesk, you also need to authorize to HelpDesk. Could you check how other companies and products phrase it?

Reply:
1. I like how Meta describes it because it is short. Twitter, for example, makes it more descriptive - `Authentication management`- so we could write: `How does HelpDesk authorization management look like?' 
And below we could write: '(...)with LiveChat. To get more details about HelpDesk authentication or authorization management, take a look at:...`
WDYT?

Reply:
idealnie - łatwo i zrozumiale :)